.github/workflows/add_new_release_to_pongo.yml

```diff
@@ -0,0 +1,125 @@
+name: Add New Release to Pongo
+
+on:
+  push:
+    tags:
+    - '[1-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  set_vars:
+    name: Set Vars
+    runs-on: ubuntu-latest
+    outputs:
+      ce_ee: ${{ steps.define_vars.outputs.CE_EE }}
+      tag_version: ${{ steps.define_vars.outputs.TAG_VERSION }}
+      pr_branch: ${{ steps.define_vars.outputs.PR_BRANCH }}
+    steps:
+    - name: Define Vars
+      id: define_vars
+      shell: bash
+      run: |
+        if [[ "${GITHUB_REPOSITORY,,}" = "kong/kong" ]] ; then
+          CE_EE=CE
+        elif [[ "${GITHUB_REPOSITORY,,}" = "kong/kong-ee" ]] ; then
+          CE_EE=EE
+        fi
+        echo "CE_EE=$CE_EE" >> "$GITHUB_OUTPUT"
+
+        if [[ "${{ github.event_name }}" == "push" ]] ; then
+          TAG_VERSION="${{ github.ref_name }}"
+        elif [[ "${{ github.event_name }}" == "release" ]] ; then
+          TAG_VERSION="${{ github.event.release.tag_name }}"
+        fi
+        echo "TAG_VERSION=$TAG_VERSION" >> "$GITHUB_OUTPUT"
+
+        echo "PR_BRANCH=add-${CE_EE,,}-version-${TAG_VERSION}" >> "$GITHUB_OUTPUT"
+  add_release_to_pongo:
+    name: Add Release to Pongo
+    runs-on: ubuntu-latest
+    needs:
+    - set_vars
+    env:
+      GITHUB_TOKEN: ${{ secrets.PAT }}
+    steps:
+    - name: Checkout Pongo
+      id: checkout_pongo
+      uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.PAT }}
+        repository: kong/kong-pongo
+        ref: master
```

Comment:
once we checked out Pongo, we should be able to use the `assets/add_version.sh` script from Pongo. Note that the exitcode is 0, 1, or 2. 2 means that the version already existed in Pongo. So we should ensure that the workflow will not fail on exitcode 2, it should report success, to not fail the CI task.

Reply:
Yep, let's review the EE PR first as we can test it internally before publishing it here.